source/index-management/tests/README.rst

```diff
@@ -155,37 +160,38 @@ Case 4: Driver can update a search index
 #. Create a collection with the "create" command using a randomly generated name (referred to as ``coll0``).
 #. Create a new search index on ``coll0`` with the following definition:
 
-  .. code:: typescript
+   .. code:: typescript
 
-    {
-      name: 'test-search-index',
-      definition: {
-        mappings: { dynamic: false }
-      }
-    }
+     {
+       name: 'test-search-index',
+       definition: {
+         mappings: { dynamic: false }
+       }
+     }
 
 #. Assert that the command returns the name of the index: ``"test-search-index"``.
 #. Run ``coll0.listSearchIndexes()`` repeatedly every 5 seconds until the following condition is satisfied:
-   1. An index with the ``name`` of ``test-search-index`` is present and index has a field ``queryable`` with the value of ``true``.
+
+   - An index with the ``name`` of ``test-search-index`` is present and index has a field ``queryable`` with the value of ``true``.
 
 #. Run a ``updateSearchIndex`` on ``coll0``, using the following definition.
-  
-  .. code:: typescript
 
-    {
-      name: 'test-search-index',
-      definition: {
-        mappings: { dynamic: true }
-      }
-    }
+   .. code:: typescript
+
+     {
+       name: 'test-search-index',
+       definition: {
+         mappings: { dynamic: true }
+       }
+     }
 
 #. Assert that the command does not error and the server responds with a success.
-#. Run ``coll0.listSearchIndexes()`` repeatedly every 5 seconds until the following condition is satisfied:
-   1. An index with the ``name`` of ``test-search-index`` is present.  This index is referred to as ``index``.
-   2. The index has a field ``queryable`` with a value of ``true`` and has a field ``status`` with the value of ``READY``.
-  
-#. Assert that an index is present with the name ``test-search-index`` and the definition has a
-  property ``latestDefinition`` whose value is ``{ 'mappings': { 'dynamic': true } }``.
+#. Run ``coll0.listSearchIndexes()`` repeatedly every 5 seconds until the following conditions are satisfied:
+
+   - An index with the ``name`` of ``test-search-index`` is present.  This index is referred to as ``index``.
+   - The index has a field ``queryable`` with a value of ``true`` and has a field ``status`` with the value of ``READY``.
+
+#. Assert that an index is present with the name ``test-search-index`` and the definition has a property ``latestDefinition`` whose value is ``{ 'mappings': { 'dynamic': true } }``.
```

Comment:
Alternatively, this could have been fixed by adjusting the wrapping on the second line; however, other steps in this file used a single line so I did so here for consistency.